rai/results/result_table_test.go

```
@@ -0,0 +1,331 @@
+        },
+        {
+            "tag": "CONSTANT_TYPE",
+            "constantType": {
+                "relType": {
+                    "tag": "PRIMITIVE_TYPE",
+                    "primitiveType": "INT_64"
+                },
+                "value": {
+                    "arguments": [
+                        {
+                            "tag": "INT_64",
+                            "int64Val": "1"
+                        }
+                    ]
+                }
+            }
+        },
+        {
+            "tag": "CONSTANT_TYPE",
+            "constantType": {
+                "relType": {
+                    "tag": "PRIMITIVE_TYPE",
+                    "primitiveType": "STRING"
+                },
+                "value": {
+                    "arguments": [
+                        {
+                            "tag": "STRING",
+                            "stringVal": "Zm9v"
+                        }
+                    ]
+                }
+            }
+        },
+        {
+            "tag": "PRIMITIVE_TYPE",
+            "primitiveType": "STRING"
+        },
+        {
+            "tag": "CONSTANT_TYPE",
+            "constantType": {
+                "relType": {
+                    "tag": "PRIMITIVE_TYPE",
+                    "primitiveType": "STRING"
+                },
+                "value": {
+                    "arguments": [
+                        {
+                            "tag": "STRING",
+                            "stringVal": "YmFy"
+                        }
+                    ]
+                }
+            }
+        },
+        {
+            "tag": "PRIMITIVE_TYPE",
+            "primitiveType": "CHAR"
+        },
+        {
+            "tag": "PRIMITIVE_TYPE",
+            "primitiveType": "INT_64"
+        }
+    ]
+}
+`
+
+func makeMetadata(json string) pb.RelationId {
+	var metadata pb.RelationId
+	err := protojson.Unmarshal([]byte(json), &metadata)
+	if err != nil {
+		panic(err)
```

this should also return the error